src/functions_framework/__init__.py

```diff
@@ -256,9 +256,15 @@ def handle_none(rv):
 
     # Extract the target function from the source file
     if not hasattr(source_module, target):
```

Comment:
Would you mind syncing this PR to main branch? This line should now be in [_function_registry.py](https://github.com/GoogleCloudPlatform/functions-framework-python/blob/master/src/functions_framework/_function_registry.py#L37).

Reply:
@KaylaNguyen I'm not a maintainer of this repo anymore, and thus don't have write access to this branch.

Reply:
Hi Dustin, would you mind forking this repo and copy this branch to a new one on di/functions-framework-python and create a PR from that instead? Then you won't need to be a maintainer of the repo to make changes to the branch. We can add you as maintainer of this repo to proceed with these 2 commits but all admins are OOO atm.